parsers/FR_O.py

```diff
@@ -203,3 +213,33 @@ def fetch_production(
             }
         )
     return return_list
+
+
+def fetch_price(
+    zone_key: str,
+    session: Optional[Session] = None,
+    target_datetime: Optional[datetime] = None,
+    logger=getLogger(__name__),
+):
+    data_objects, date_string = fetch_data(zone_key, session, target_datetime, logger)
+
+    return_list: List[Dict[str, Any]] = []
+    for data_object in data_objects:
+        price: Union[float, int, None] = None
+        for mode_key in data_object:
+            if mode_key in PRICE_MAPPING:
+                if price == None:
+                    price = data_object[mode_key]
+                else:
+                    price += data_object[mode_key]
```

Comment:
```suggestion
                price = data_object[mode_key]
```
There should only be one instance of price data per datapoint if I remember correctly so we can just directly assign the data without any issue.